src/pickledb.rs

```diff
@@ -518,7 +518,7 @@ impl PickleDb {
         V: DeserializeOwned,
     {
         match self.map.get(key) {
-            Some(val) => self.serializer.deserialize_data::<V>(&val),
+            Some(val) => self.serializer.deserialize_data::<V>(val),
```

Comment:
The clippy issues were already resolved in https://github.com/seladb/pickledb-rs/pull/25

Reply:
Ah, nice.  Rebased